department-of-veterans-affairs/va.gov-cms
# VAMC-11521: VISN 23 systems menus.

## Description

Map out and Create Admin and Editorial Menus for VISN 23 Systems.

## QA steps

As admin

- [ ]  Visit <Manage/content/menus/VAMC system menus> and validate that menus displayed match this.
<img width="1680" alt="Screen Shot 2020-08-07 at 8 44 06 AM" src="https://user-images.githubusercontent.com/56320995/89647481-a0ea8800-d88b-11ea-947e-b7ec0c28d6c6.png">


Visit the following

- [ ]   /admin/structure/types/manage/health_care_local_facility
- [ ]   /admin/structure/types/manage/health_care_local_health_service
- [ ]   /admin/structure/types/manage/health_care_region_page
- [ ]  /admin/structure/types/manage/regional_health_care_service_des
- [ ]  /admin/structure/types/manage/vamc_operating_status_and_alerts
and validate that the content types have new Menu items (VISN 23 systems) selected as enabled (check box) to match this.

<img width="1680" alt="Screen Shot 2020-08-07 at 8 47 35 AM" src="https://user-images.githubusercontent.com/56320995/89647572-c081b080-d88b-11ea-8e9c-e1e518064c7b.png">


- [ ]  Visit "Sections/veterans health administration/VAMC facilities" and validate that menus displayed for VISN 23 match this.

<img width="1680" alt="Screen Shot 2020-08-07 at 8 49 44 AM" src="https://user-images.githubusercontent.com/56320995/89647611-d3948080-d88b-11ea-9689-38c6dd883102.png">

- [ ]  Visit "Manage/Content/Menus/CMS administration/Administration" and validate that VISN 23 systems are assigned as child.
<img width="1680" alt="Screen Shot 2020-08-07 at 8 52 29 AM" src="https://user-images.githubusercontent.com/56320995/89647666-e5762380-d88b-11ea-87e6-40586ecfff78.png">

## Definition of Done
